user-operator/src/main/java/io/strimzi/operator/user/model/acl/SimpleAclRule.java

```diff
@@ -118,6 +121,118 @@ public String toString() {
+                break;
+            case ALTERCONFIGS:
+                kafkaOperation = org.apache.kafka.common.acl.AclOperation.ALTER_CONFIGS;
+                break;
+            case DESCRIBECONFIGS:
+                kafkaOperation = org.apache.kafka.common.acl.AclOperation.DESCRIBE_CONFIGS;
+                break;
+            case IDEMPOTENTWRITE:
+                kafkaOperation = org.apache.kafka.common.acl.AclOperation.IDEMPOTENT_WRITE;
+                break;
+            case ALL:
+                kafkaOperation = org.apache.kafka.common.acl.AclOperation.ALL;
+                break;
+            default:
+                throw new IllegalArgumentException("Invalid Acl operation: " + operation);
+        }
+
+        return new AccessControlEntry(principal.toString(), getHost(), kafkaOperation, kafkaType);
+    }
+
+    public AccessControlEntryFilter toKafkaAccessControlEntryFilter(KafkaPrincipal principal)   {
+        AclPermissionType kafkaType;
+        org.apache.kafka.common.acl.AclOperation kafkaOperation;
+
+        switch (type) {
+            case DENY:
+                kafkaType = AclPermissionType.DENY;
+                break;
+            case ALLOW:
+                kafkaType = AclPermissionType.ALLOW;
+                break;
+            default:
+                throw new IllegalArgumentException("Invalid Acl type: " + type);
+        }
+
+        switch (operation) {
+            case READ:
+                kafkaOperation = org.apache.kafka.common.acl.AclOperation.READ;
+                break;
+            case WRITE:
+                kafkaOperation = org.apache.kafka.common.acl.AclOperation.WRITE;
+                break;
+            case CREATE:
+                kafkaOperation = org.apache.kafka.common.acl.AclOperation.CREATE;
+                break;
+            case DELETE:
+                kafkaOperation = org.apache.kafka.common.acl.AclOperation.DELETE;
+                break;
+            case ALTER:
+                kafkaOperation = org.apache.kafka.common.acl.AclOperation.ALTER;
+                break;
+            case DESCRIBE:
+                kafkaOperation = org.apache.kafka.common.acl.AclOperation.DESCRIBE;
+                break;
+            case CLUSTERACTION:
+                kafkaOperation = org.apache.kafka.common.acl.AclOperation.CLUSTER_ACTION;
+                break;
+            case ALTERCONFIGS:
+                kafkaOperation = org.apache.kafka.common.acl.AclOperation.ALTER_CONFIGS;
+                break;
+            case DESCRIBECONFIGS:
+                kafkaOperation = org.apache.kafka.common.acl.AclOperation.DESCRIBE_CONFIGS;
+                break;
+            case IDEMPOTENTWRITE:
+                kafkaOperation = org.apache.kafka.common.acl.AclOperation.IDEMPOTENT_WRITE;
+                break;
+            case ALL:
+                kafkaOperation = org.apache.kafka.common.acl.AclOperation.ALL;
+                break;
+            default:
```

Comment:
Again, there's an `ANY` which is probably _meant_ to be handled in the same way as the default case, but it would be good to be explicit.

Reply:
Ditto as above, `ANY` is not a valid value in our corresponding API.